swap_meet/vendor.py

```diff
@@ -1,2 +1,123 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+    
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        self.inventory.remove(item)
+        return item
+    
+
+    def get_by_id(self, id):
+        for item in self.inventory:
```

Comment:
This is well done too! You could also initialize a return variable like 
```python
result = None
```
then assign the `item` to it if there's a matching `id`, finally returning that `result`. Then you wouldn't have to return from within the loop.